scaffoldmaker/meshtypes/meshtype_3d_colon1.py

```diff
@@ -157,23 +178,34 @@ def generateBaseMesh(region, options):
         :return: annotationGroups
         """
         centralPath = options['Central path']
-        elementsCountAroundTC = options['Number of elements around tenia coli']
-        elementsCountAroundHaustrum = options['Number of elements around haustrum']
-        elementsCountAround = (elementsCountAroundTC + elementsCountAroundHaustrum)*3
-        elementsCountAlongHaustrum = options['Number of elements along haustrum']
-        elementsCountThroughWall = options['Number of elements through wall']
-        haustraSegmentCount = options['Number of haustra segments']
-        radius = options['Inner radius']
-        cornerInnerRadiusFactor = options['Corner inner radius factor']
-        haustrumInnerRadiusFactor = options['Haustrum inner radius factor']
-        haustrumLengthEndDerivativeFactor = options['Haustrum length end derivative factor']
-        haustrumLengthMidDerivativeFactor = options['Haustrum length mid derivative factor']
-        widthTC = options['Tenia coli width']
```

Comment:
Make consistent with other variables, where width, thickness is last, also with naming conventions I use in heart (including variables starting with lower case, types with upper case):
tcWidth
tcThickness